resources/views/livewire/custom-field-set-default-values-for-model.blade.php

```diff
@@ -1,54 +1,79 @@
                                             <span class="input-group-addon"><i class="fas fa-calendar" aria-hidden="true"></i></span>
                                         </div>
                                     </div>
+
                                 @elseif ($field->element == "text")
-                                    <input class="form-control m-b-xs" type="text" value="{{ $field->defaultValue($model_id) }}" id="default-value{{ $field->id }}" name="default_values[{{ $field->id }}]">
+
+
+                                        <input class="form-control" type="text" value="{{ $field->defaultValue($model_id) }}" id="default-value{{ $field->id }}" name="default_values[{{ $field->id }}]">
+
+
                                 @elseif($field->element == "textarea")
-                                    <textarea class="form-control" style="width: 100%;" id="default-value{{ $field->id }}" name="default_values[{{ $field->id }}]">{{ $field->defaultValue($model_id) }}</textarea><br>
+
+
+                                        <textarea class="form-control" style="width: 100%;" id="default-value{{ $field->id }}" name="default_values[{{ $field->id }}]">{{ $field->defaultValue($model_id) }}</textarea>
+
+
                                 @elseif($field->element == "listbox")
 
-                                    <select class="form-control m-b-xs" name="default_values[{{ $field->id }}]">
-                                        <option value=""></option>
-                                        @foreach(explode("\r\n", $field->field_values) as $field_value)
-                                            <option value="{{$field_value}}" {{ $field->defaultValue($model_id) == $field_value ? 'selected="selected"': '' }}>{{ $field_value }}</option>
-                                        @endforeach
-                                    </select>
+
+                                        <select class="form-control" name="default_values[{{ $field->id }}]">
+                                            <option value=""></option>
+                                            @foreach(explode("\r\n", $field->field_values) as $field_value)
+                                                <option value="{{$field_value}}" {{ $field->defaultValue($model_id) == $field_value ? 'selected="selected"': '' }}>{{ $field_value }}</option>
+                                            @endforeach
+                                        </select>
+
+
                                 @elseif($field->element == "radio")
+
                                     @foreach(explode("\r\n", $field->field_values) as $field_value)
-                                        <input type='radio' name="default_values[{{ $field->id }}]" value="{{$field_value}}" {{ $field->defaultValue($model_id) == $field_value ? 'checked="checked"': '' }} />{{ $field_value }}<br />
+                                    <label class="col-md-3 form-control" for="{{ str_slug($field_value) }}">
```

Comment:
It doesn't seem that *likely* but I think it'd be possible to duplicate these `for="blah"` values throughout different sets of checkboxes/radio buttons. And also possibly duplicate the `id` values for some of the checbox/radio buttons. Also not fatal by any means, but I think we could possibly skip these `for/id` associations entirely - since the `<label>` actually wraps the `<input>` element.

Reply:
It wraps the input element, but it doesn't behave the way you'd expect. I can keep playing with it